README.md

```diff
@@ -1 +1,182 @@
-# Catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Catalog-of-my-things ](#-practice-test-driven-developmenttdd-)
+  - [🛠 Built With ](#-built-with-)
+  - [🛠 Tech Stack ](#-tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors](#-authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog-of-my-things <a name="about-project"></a>
+
+We developed a console application for tracking various items you have acquired such as books, music albums, movies, and games. The UML class diagram provided the foundation for the project. The data is stored in JSON files, and we have also created a database with table structures that are similar to the program's class structure.
+
+## 🛠 Built With <a name="built-with"></a>
+
+## 🛠 Tech Stack <a name="tech-stack"></a>
+
+  <li><a href="">Ruby</a></li>
+  <li><a href="">Rspec</a></li>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Add A Book.
+> - Add a Music Albums.
+> - Add a Game.
+> - List all items added.
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Check the link](https://drive.google.com/file/d/1sA-uJ4UwCbzWPpQ_cmUsSQp0tStwKuup/view?usp=sharing)

```

Comment:
- [ ] For this project, it's required that all members of the project appear on the video and must contribute to the presentation. It's only one member that presented the whole project. Kindly, let all three group members present 👍🏽
- [ ] Please be sure the Maximum length of the video is 5 minutes. Currently, your video exceeds 5 minutes. 👍🏽
- [ ] When presenting the video, you don't have to show all your code files or all the code implementation. Kindly introduce the project and team members introduce themselves, talk about the section each member worked on and then go straight to the demo. In the end, mention a small piece of code that you are proud of implementing. 👍🏽

<img width="1049" alt="Screenshot 2023-03-24 at 11 29 09 AM" src="https://user-images.githubusercontent.com/50941074/227510021-959d5835-1809-4261-8ddd-47f3d0f6f291.png">

NOTE: the max length of the video should not exceed 5 minutes.